README.md

```diff
@@ -27,9 +27,17 @@ Usage in python
 you need the environment define by crop_env.yml. I suggest to use anaconda or miniconda as environment manager
 
 ```
-python Crop_image_main.py INHS_FISH_000742.jpg INHS_FISH_000742.json INHS_FISH_000742_cropped.jpg
+Crop_image_main.py [-h] [--increase] input_image imput_metadata output
 ```
 
+Example using the Test_data with 5% increase in size
+
+```
+Crop_image_main.py INHS_FISH_000742.jpg INHS_FISH_000742.json INHS_FISH_000742_cropped.jpg --increase 0.05
+```
+
+The `increase` parameter defines the percentage increase of the bounding box from the original metadata size in percentage for each direction (5% increase in widht and 5% in height as 0.05)
```

Comment:
```suggestion
The `increase` parameter defines the percentage increase of the bounding box from the original metadata size in percentage for each direction (5% increase in width and 5% in height as 0.05)
```

Reply:
Why not show an example with a different increase amount since `0.05` is the default?

Reply:
I would also add a comment answering this question from the issue:
```
Also, when calculating the pixels to add in each direction, are fractions subject to floor or ceiling?
```
From what I can tell the answer is floor since we are using `int()` which truncates towards zero:
https://github.com/hdr-bgnn/Crop_image/blob/e52c367dde6cce08c4accdd0f4f92ce4eb04e47d/Crop_image_main.py#L79-L80

Reply:
yes it is floor(). May be we could use round() which would convert to the closest integer, but ultimately this is minor since in this case or the other we lose or gain a pixel or two which won't change much the cropping.

Reply:
@thibaulttabarin I don't think you've addressed this yet. Specifically, the following remains unaddressed (and would be trivial to implement):
> Why not show an example with a different increase amount since 0.05 is the default?

Also, another comment suggests to document the fact that the calculation of the number of pixels to add is subject to integer truncation (which, BTW, strictly speaking [is _not_ the same as `floor()`](https://en.wikipedia.org/wiki/Floor_and_ceiling_functions): floor(-2.4) == 3, including in Python; that said, for positive integers it is the same).

Reply:
Change the increase value in the documentation example 
https://github.com/hdr-bgnn/Crop_image/pull/5/commits/b53cbd2c629c22563d8bf96d81fada235deb3652